package/com.unity.formats.usd/Tests/Editor/EditorTests.cs

```diff
@@ -9,5 +12,35 @@ public void DummyTest()
         {
         }
     }
+    
+    public class ScopeLoadingTest
+    {
+        private GameObject usdRoot;
+        private string filepath = "test_#154/test_collections.usda";
+        
+        [SetUp]
+        public void SetUp()
+        {
+            InitUsd.Initialize();
+            var stage = pxr.UsdStage.Open(Path.Combine(Application.dataPath, filepath), pxr.UsdStage.InitialLoadSet.LoadNone);
+            var scene = Scene.Open(stage);
+            usdRoot = USD.UsdMenu.ImportSceneAsGameObject(scene);
+            scene.Close();
+        }
+        
+        [Test]
+        public void TestScopeWithoutChildren()
+        {
+            var scopeWithChildren = usdRoot.transform.Find("TestComponent/geom");
+            Assert.IsNotNull(scopeWithChildren);
```

Comment:
Do we want to assert anything about the data on the scope? E.g. that it has a UsdPrim component or that the transform is identity?

Reply:
@jcowles, is it valuable for this test? Or should they be in separate ones?

Reply:
You're right, I'll do both and add a little bit more because I'm realizing the test doesn't do much. Looks like I lost some bits between stashes...